### DISCUSSION TITLE
Canais YouTube sobre Programação

### DISCUSSION THREAD

**Discussion by willianjusten** [frontendbr/forum#1013] [Dicas]
*Eita que a issue foi criada antes de eu escrever LOL*

Enfiiim, eu hoje criei meu [humilde canal no Youtube](http://youtube.com/WillianJustenCursos) e tenho muiiito que aprender de tudo.

Então queria dicas de canais que vocês conhecem e acham bons e até mesmo que estilo de vídeos vocês acham que é interessante. 

Eu ia colocar uma lista de canais que eu já sigo, mas o YouTube tá fora do ar =o

Mas uns que vejo bastante:

- Loiane Grogner
- Fun Fun Function
- Balta.io
- CSS Cast Show
- BrazilJS
- The Coding Train
- Rodrigo Branas

**Comment by raulfernando** [comment]
Curto muito o do  @vedovelli

**Comment by ghost** [comment]
Atualizei com os que eu conheço. 

BRs
[Bruno Germano](https://www.youtube.com/channel/UCBWbWViVqDHckknir8PIIdg)
[AndroidPro](https://www.youtube.com/channel/UCnhwRB19LnYt5WviCKbVQwQ)
[coisadedev](https://www.youtube.com/channel/UC4W-b9Q1I0mzGGzFKOu-66g)
[ZeroBugs](https://www.youtube.com/channel/UC6BN1oSRAxcABupjn_bnT7g)

Gringos
[Devtips](https://www.youtube.com/channel/UCyIe-61Y8C4_o-zZCtO4ETQ)
[Codecourse](https://www.youtube.com/channel/UCpOIUW62tnJTtpWFABxWZ8g)
[Codedamn](https://www.youtube.com/channel/UCJUmE61LxhbhudzUugHL2wQ)
[Designmodo](https://www.youtube.com/channel/UCLOJhs8F2y2iYthajewsE3w)
[Quentin Watts](https://www.youtube.com/channel/UCtGGGu_hp8ToQ9BR6Vni19w)
[School Of Net](https://www.youtube.com/channel/UCIlafifr-E57jct9knCrZzw)
[Brad Hussey](https://www.youtube.com/channel/UCVguiojKA6iobcySMJ5boNA)
[LevelTutsUp](https://www.youtube.com/user/LevelUpTuts)
[Alessandro Castellani](https://www.youtube.com/channel/UCbmBY_XYZqCa2G0XmFA7ZWg)
[CodingPhase](https://www.youtube.com/channel/UC46wWUso9H5KPQcoL9iE3Ug)
[Peter Fisher](https://www.youtube.com/user/howtocodewell)

**Comment by willianjusten** [comment]
Opa boa @raulfernando , havia me esquecido de colocar ele na lista. Eu curto também =)
E @daniilomello é o YouTube que tá dando erro 500 em páginas sem cache, esse link aí que tu botou é do meu usuário e não do canal de fato.

**Comment by doug2k1** [comment]
Gosto de vídeos curtos. Se o assunto for complexo, que seja dividido em varias partes.

Alguns que sigo:

DevPleno: https://www.youtube.com/channel/UC07JWf9A0B1scApbS1Te7Ww

Wes Bos: https://www.youtube.com/user/wesbos

LevelUp Tuts: https://www.youtube.com/user/LevelUpTuts

Reactcasts: https://www.youtube.com/channel/UCZkjWyyLvzWeoVWEpRemrDQ

**Comment by rodrigocode4** [comment]
Curso em Vídeo é um ótimo canal de ensino de programação.
https://www.youtube.com/user/cursosemvideo

**Comment by tunnes** [comment]
Eu acompanhava bastante o [Fun Fun Function](https://www.youtube.com/channel/UCO1cgjhGzsSYb1rsB4bFe4Q) e o [canal](https://www.youtube.com/channel/UCU5JicSrEM5A63jkJ2QvGYw) do @filipedeschamps

**Comment by crisgon** [comment]
Uma pena que tá parado faz tempo, mas o canal do [Estevan Maito](https://www.youtube.com/channel/UCIUOmAsotrzdK8VsZuHfVHw) tem muito conteúdo.
Tem também o [Giraffe Academy](https://www.youtube.com/channel/UCvmINlrza7JHB1zkIOuXEbw)

**Comment by ghost** [comment]
linux tips e web school

**Comment by ivosoares** [comment]
## Português
- [CODE CAST](https://www.youtube.com/channel/UCTluPqMkm90zyw6mCde561A/videos)
- [Waldemar Neto](https://www.youtube.com/user/waldemaneto/videos)
- [iMasters](https://www.youtube.com/user/imasters)
- [Ricardo Sanches](https://www.youtube.com/user/RicVSanches/videos) Design/Técnologia
- [Tiago Matos](https://www.youtube.com/user/tiagomatosweb) Vue.js
- [Leandro Bighetti](https://www.youtube.com/channel/UCKN63lTXUgCSjR5gPNDUjmw)
- [Harlley Oliveira](https://www.youtube.com/user/CursoDeProgramacao/videos)
- [O Programador](https://www.youtube.com/channel/UCa4Dj04ABMCxBUJ_aWTP9Bg/videos)
- [Pagar.me Talks](https://www.youtube.com/channel/UCNhSCufrcOMeFvzEM7tt9Lw)

## Inglês
- [The NET Ninja](https://www.youtube.com/channel/UCW5YeuERMmlnqo4oq8vwUpg/featured)
- [Coding Tech](https://www.youtube.com/channel/UCtxCXg-UvSnTKPOzLH4wJaQ)
- [Alessandro Castellani](https://www.youtube.com/user/williamprey/featured) Wordpress
- [Vue Mastery](https://www.youtube.com/channel/UCa1zuotKU4Weuw_fLRnPv0A/videos) Vue.js/JS

**Comment by Cleysonlb** [comment]
Ultimamente eu to viciado em acompanhar canais de programadores/programadoras que mostram seu dia a dia, o que eles chama de `a day in the life of a software engineer` vou citar alguns que acompanho:

- [Hello Mayuko](https://www.youtube.com/user/hellomayuko/videos)
- [Joma Tech](https://www.youtube.com/channel/UCV0qA-eDDICsRR9rPcnG7tw)
- [ENGINEERED TRUTH](https://www.youtube.com/user/EngineeredTruth)
- [Alexa & Luba](https://www.youtube.com/watch?v=22ggFtntSHg)
- [Jarvis Johnson](https://www.youtube.com/user/VSympathyV)

**Comment by mylenaleiteb** [comment]
Tem um canal que eu gosto muito, é de um casal que fala sobre vários assuntos de programação [Código Fonte TV](https://www.youtube.com/channel/UCFuIUoyHB12qpYa8Jpxoxow) e o [site](https://www.codigofonte.com.br/) deles

**Comment by ojvribeiro** [comment]
Tem alguns canais brasileiros e gringos que já acompanho faz um tempo e gosto muito do conteúdo deles.

Brasileiros
----
- [Filipe Deschamps](https://www.youtube.com/channel/UCU5JicSrEM5A63jkJ2QvGYw)
- [Fábio Akita](https://www.youtube.com/channel/UCib793mnUOhWymCh2VJKplQ)
- [Rocketseat](https://www.youtube.com/channel/UCSfwM5u0Kce6Cce8_S72olg)
- [Código Fonte TV](https://www.youtube.com/channel/UCFuIUoyHB12qpYa8Jpxoxow)

Gringos
----
- [Dani Krossing](https://www.youtube.com/channel/UCzyuZJ8zZ-Lhfnz41DG5qLw) (antigo **mmtuts**)
- [thenewboston](https://www.youtube.com/channel/UCJbPGzawDH1njbqV-D5HqKw)

**Comment by renedocket** [comment]
Pra ir completando a listinha gosto desses também, abordam diversos temas relevantes: 

- [Attekita Dev](https://www.youtube.com/channel/UCetRsdZxDQDcgVDJd6erz6g)
- [CollabCode](https://www.youtube.com/c/CollabCode/videos)
- [Erick Wendel](https://www.youtube.com/channel/UCh84012dEUE076wM2CVFN9A)
- [Ana Beatriz Neri](https://www.youtube.com/channel/UCBjoWT-P17Bl66D52RwqdGA?view_as=subscriber)
- [Felipe Fialho](https://www.youtube.com/channel/UCHf-3DHd-VYn7HmddjfBj0A)
- [Diolinux](https://www.youtube.com/channel/UCEf5U1dB5a2e2S-XUlnhxSA)
